api/config/config_staging.json

```diff
@@ -110,8 +110,9 @@
   },
   "actionAudit": {
     "logName": "workbench-action-audit-staging",
-    "bigQueryDataset": "workbench_action_audit_staging",
-    "bigQueryTable": "workbench_action_audit_staging"
+    "bigQueryDataset": "workspace_action_audit_staging",
```

Comment:
somehow this was configured as work**space** not work**bench**.  I suspect auditing doesn't work in Staging either.